src/main/thrift/parquet.thrift

```diff
@@ -511,6 +511,51 @@ struct ColumnMetaData {
    * This information can be used to determine if all data pages are
    * dictionary encoded for example **/
   13: optional list<PageEncodingStats> encoding_stats;
+
+  /** Byte offset from beginning of file to bloom filter data**/
+  14: optional i64 bloom_filter_offset;
+}
+
+/**
+  * Definition of bloom filter algorithm.
+  */
+enum BloomFilterAlgorithm {
+  /** Block based bloom filter. 
+   * The bloom filter bitset is separated into tiny bucket as tiny bloom 
+   * filter, the high 32 bits hash value is used to select bucket, and 
+   * lower 32 bits hash values are used to set bits in tiny bloom filter.
+   * See “Cache-, Hash- and Space-Efficient Bloom Filters”. Specifically, 
+   * one tiny bloom filter contains eight 32-bit words and the algorithm
```

Comment:
Looks good. Also explain the endianness?

Reply:
The comments of hash say take plain encoding of content. We don't have to explain endianness here again, since algorithm itself is endianness irrelevant.

Reply:
Maybe explain the endianness of how each 4-byte word is stored?